### DISCUSSION TITLE
Hacktoberfest2k24 Discussion

### DISCUSSION THREAD

**Discussion by sayan0328** [sudo-parnab/Its-apptober#15] [General]
I have done 3 PR but the verification takes a long time, will it count the PRs verified after 31st October?

**Comment by Parnab03** [comment]
Pull requests (PRs) submitted for Hacktoberfest undergo a 7-day review period by the Hacktoberfest team for verification. **If you raise a PR before October 31 and it is accepted by the repository maintainer, it will count towards your Hacktoberfest contributions**. Even if the verification period extends into the first week of November, as long as the PR was submitted before the deadline, it will still be considered valid.

**Comment by Parnab03** [comment]
Take a look: https://hacktoberfest.com/participation/#pr-mr-details

**Comment by sayan0328** [comment]
"PR/MRs that are passing all checks and are still in the review period on October 31 can continue the seven-day review period into November." Got it, thanks!
